imqsauth/http.go

```diff
@@ -597,14 +620,19 @@ func httpHandlerUpdateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	firstname := strings.TrimSpace(r.http.URL.Query().Get("firstname"))
 	lastname := strings.TrimSpace(r.http.URL.Query().Get("lastname"))
 	mobilenumber := strings.TrimSpace(r.http.URL.Query().Get("mobilenumber"))
+	strAuthusertype := strings.TrimSpace(r.http.URL.Query().Get("authusertype"))
+	authUserType, err := strconv.Atoi(strAuthusertype)
+	if err != nil {
```

Comment:
Need to verify here that the AuthUserType is indeed a valid one, not just a valid integer.